tests/test_updater.py

```diff
@@ -283,3 +308,121 @@ def test_update_logging(update_result, delta_logger, records_data_w_subjects):
     assert "" == r1_log_entry["error"]
     r1_delta = "http://example.org/foo/0 -> X"
     assert r1_delta == r1_log_entry["deltas"]
+
+
+def test_update_keep_trace(
+    create_subject_data, minimal_record_input, create_record_data_fn,
+):
+    # Test that keep_trace template is placed in keep_trace field
+    # if subject delta op is marked as keep_trace
```

Comment:
In future PR we'll settle what values of keep_trace in the csv will be truthy / what values will be falsy.